### PR TITLE
Make some small fixes

### PR DESCRIPTION
**Pull request by mjmayank**
a small PR as a test to learn the codebase a little bit

### CODE REVIEW

File: src/Screens/Explore/components/DiveSite.tsx [hunk 1]
```diff
@@ -79,12 +79,13 @@ const DiveSite: FunctionComponent<DiveSiteProps> = props => {
 
   if (props.onPressContainer && typeof props.onPressContainer === 'function') {
     return (
-      <TouchableWithoutFeedback
```
Comment (Nnanyielugo) [inline src/Screens/Explore/components/DiveSite.tsx:82]: The effects of this Touchable can get a little unsightly. Are you sure you want it?

Comment (mjmayank) [reply src/Screens/Explore/components/DiveSite.tsx:82]: i tried it out and it seemed fine in the iOS simulator. TouchableHighlight seemed unsightly, is that what you're referring to?

Comment (mjmayank) [reply src/Screens/Explore/components/DiveSite.tsx:82]: i also changed the opacity to 0.8, because the default was 0.2

Comment (Nnanyielugo) [reply src/Screens/Explore/components/DiveSite.tsx:82]: I think changing the opacity would do the trick. Approving. now.

**Pull request merged**